src/components/ComplaintDetail/ComplaintDetail.less

```diff
@@ -116,7 +111,7 @@
     .subitem {
       margin-right: @gutter-minimum;
 
-      font-family: @webfont-demi;
+      font-family: 'AvenirNextLTW01-Medium', sans-serif;
```

Comment:
If you're using https://github.com/cfpb/design-system/blob/8b5f6f431fdc7d3bf41ff7f93d67c187455a516b/packages/cfpb-typography/src/licensed-fonts.less, than this should be `font-family: 'Avenir Next', sans-serif;` and then bolded (to use Avenir Next Medium instead of Regular).

Same with other `font-family` references.